pkg/apis/release/v1alpha1/types.go

```diff
@@ -122,6 +124,20 @@ type PayloadCoordinates struct {
 	ImagestreamTagName string `json:"imagestreamTagName,omitempty"`
 }
 
+// PayloadCreationConfig the configuration used to create the ReleasePayload
+type PayloadCreationConfig struct {
+	// ReleaseCreationConfig houses the configuration of the release creation job
+	ReleaseCreationConfig ReleaseCreationConfig `json:"releaseCreationConfig,omitempty"`
```

Comment:
`ReleaseCreationCoordinates` to match your other coordinates?

Reply:
Done